sympy/integrals/integrals.py

```diff
@@ -399,16 +401,22 @@ def doit(self, **hints):
         if function.is_zero:
             return S.Zero
 
+        # hacks to handle special cases
         if isinstance(function, MatrixBase):
-            return function.applyfunc(lambda f: self.func(f, self.limits).doit(**hints))
+            return function.applyfunc(
+                lambda f: self.func(f, self.limits).doit(**hints))
 
         if isinstance(function, FormalPowerSeries):
```

Comment:
@leosartaj , does this look right to you? It seems odd/wrong to integrate wrt the first limit if there are multiple integrations...so now (in case this case exists) a NotImplementedError is raised if there is more than one integation, e.g. `integrate(foo, lim1, lim2)`